src/index.css

```diff
@@ -139,3 +138,7 @@ nav > * {
   float: right;
   margin-left: auto;
 }
+
+h4 {
+  font-size: large;
```

Comment:
Since you're using bootstrap, there's probably classes you can add directly rather than having to add to your css as well - just a thought!